content/release-10-18-0/edge-10-18-0-bundle/10_18_0_1.md

```diff
@@ -6,12 +6,27 @@ layout: redirect
 
 The release notes for **Cumulocity IoT Edge release 10.18.0** contain important information about all the changes, including new features, enhancements, and fixes, that have been made to both **Cumulocity IoT Edge Appliance VM** and **Cumulocity IoT Edge on Kubernetes**. 
 
-
 You can find the corresponding documentation for **Cumulocity IoT Edge Appliance VM** at https://cumulocity.com/docs/edge/edge-introduction/ and for **Cumulocity IoT Edge on Kubernetes** at https://cumulocity.com/docs/edge-kubernetes/k8-edge-introduction/.
 
 
 ### Cumulocity IoT Edge Appliance VM
-There have not been any significant improvements or fixes related to this component since the last Maintenance release.
+
+#### Upgrade Rocky Linux operating systenm version to 8.10
+
+This release upgrades the Rocky Linux operating system to 8.10 from version 8.9. For information on updating Edge version from 10.18.0.0 to 10.18.0.1, see [Updating Edge](https://cumulocity.com/docs/edge/edge-update/). For information on upgrading Edge version from 10.17, see [Migrating Edge 10.17 to 10.18](https://cumulocity.com/guides/10.18.0/edge/update/#migration-1017-to-1018).
+
+#### Fixes
+
+|<div style="width:100px">Ticket</div>|<div style="width:200px">Area</div>|Description
+|:---|:---|:---
+|CIE-5191|Diagnostic utility|Issue with diagnostic utility, scheduled to run periodically, generating excessive diagnostic report files filling up the disk space is resolved.
+|CIE-2263|Cumulocity Core|`/tmp/cumulocity-core-karaf` directory is used by Cumulocity Core for storing the uploaded Application and Microservices zip files before processing. This directory is monitored and recreated if necessary.
+|CIE-2380|Cumulocity Agent|Issue with the Cumulocity Agent crashing when registering a child device is resolved.
```

Comment:
```suggestion
|CIE-2380|Cumulocity IoT agent|The issue with the Cumulocity IoT agent crashing when registering a child device is resolved.
```

Reply:
We refer to it at other places in the Edge documentation as Cumulocity agent (cumulocity-agent), hence used the same.

Reply:
Hi @BeateRixen , 
we seem to be referring to this agent as "cumulocity-agent", "the agent" and "Modbus agent (called “linux-agent”)" in the existing doc :( please refer to https://cumulocity.com/docs/edge/connecting-devices-to-edge/#connecting-modbus-device 
Please suggest your preference.

Reply:
I would go with "Cumulocity agent" then.